src/main/java/testapp/adapter/http/StudentService.java

```diff
@@ -0,0 +1,52 @@
+package testapp.adapter.http;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import com.google.sitebricks.At;
+import com.google.sitebricks.client.transport.Json;
+import com.google.sitebricks.headless.Reply;
+import com.google.sitebricks.headless.Request;
+import com.google.sitebricks.http.Delete;
+import com.google.sitebricks.http.Get;
+import com.google.sitebricks.http.Post;
+import testapp.core.Student;
+import testapp.core.StudentAlreadyExistsException;
+import testapp.core.StudentDto;
+import testapp.core.StudentRepository;
+
+import java.net.HttpURLConnection;
+import java.util.List;
+
+@At("/student")
+public class StudentService {
+  private StudentRepository repository;
+
+  @Inject
+  public StudentService(StudentRepository repository) {
+    this.repository = repository;
+  }
+
+  @Get
+  public Reply<?> getStudents() {
```

Comment:
since the students will be the users in our app, I would suggest naming them users would give us a better idea about there role in the system.